config/sync/core.entity_view_display.eventinstance.default.list_teaser.yml

```diff
@@ -77,7 +77,7 @@ content:
     weight: 11
     region: content
   event_description:
-    type: text_default
+    type: basic_string
```

Comment:
Is this change intentional? Didn't see it mentioned in the PR description.

Reply:
Yes and no - It's not something i did actively, but, it has come from saving the view display, and apparantly this config was missing :) I dont see it as a problem, but good that you're awake!